<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New User</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>382</width>
     <height>176</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="newUname_label">
      <property name="text">
       <string>New Username: </string>
      </property>
      <property name="buddy">
       <cstring>newUname_lineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="newUname_lineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="newPsswd_label">
      <property name="text">
       <string>New Password: </string>
      </property>
      <property name="buddy">
       <cstring>newPsswd_lineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="newPsswd_lineEdit">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="newPasswordConfirmLabel">
      <property name="text">
       <string>New Password (confirm): </string>
      </property>
      <property name="buddy">
       <cstring>newPasswordConfirmLineEdit</cstring>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="newPasswordConfirmLineEdit">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="perms_label">
      <property name="text">
       <string>Permissions: </string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="buddy">
       <cstring>readWrite_radioButton</cstring>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="readWrite_radioButton">
        <property name="text">
         <string>Read/Write</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="readOnly_radioButton">
        <property name="text">
         <string>Read Only</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="printUsers_pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Print UserList</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>newUname_lineEdit</tabstop>
  <tabstop>newPsswd_lineEdit</tabstop>
  <tabstop>newPasswordConfirmLineEdit</tabstop>
  <tabstop>readWrite_radioButton</tabstop>
  <tabstop>readOnly_radioButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>printUsers_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>printUserList()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>212</y>
    </hint>
    <hint type="destinationlabel">
     <x>165</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>printUserList()</slot>
 </slots>
</ui>
